docs/src/install and build.md

```diff
@@ -1,31 +1,28 @@
-# Install and run Erdapfel on Windows 10+
+# How to install, build and run Erdapfel
 
-- Install [NodeJS LTS](https://nodejs.org/en/download/)
-- Install [WSL2 with Ubuntu](https://docs.microsoft.com/fr-fr/windows/wsl/install-win10) (the username can only contain characters /a-z/)
-- Clone Erdapfel in the folder of your choice (ex: "C:\www\") using [Github Desktop](https://desktop.github.com/) or the terminal: 
+(Instructions for all OS's, specific instructions for Windows 10+)
+- On Windows+, Install [WSL2 with Ubuntu](https://docs.microsoft.com/fr-fr/windows/wsl/install-win10)
+- Clone Erdapfel in the folder of your choice using [Github Desktop](https://desktop.github.com/) or the terminal: 
 
 ```
 sudo apt install git
 git clone https://github.com/Qwant/erdapfel.git
 ```
 
-- Open a Ubuntu bash in this folder (shift + right-click > Open Linux CLI here)
-- Install [nvm](https://github.com/nvm-sh/nvm#install--update-script), then node 12 (`nvm install 12` + `nvm use 12`), then update npm (`npm install -g npm@7`)
-- Install tools, libraries:
+- Open a terminal in the "erdapfel" folder (on Windows: shift + right-click > Open Linux CLI here)
+- Install [nvm](https://github.com/nvm-sh/nvm#install--update-script), then node 12 (`nvm install 12` + `nvm use 12`), then npm 7 (`npm install -g npm@7`)
+- Install the project's tools and libraries:
 
 ```
 sudo apt update && sudo apt install -y build-essential git libexpat1-dev libssl-dev zlib1g-dev \
```

Comment:
I don't know why it's there, and as a Linux or iOS web developer who is a new comer to the project, I would definitely not do that without knowing how it is useful to a "simple" NodeJS project.
Do you remember what it's for on W10?

Reply:
no - it's possible that they're just old requirements for the Erdapfel from 2018.
So, if wsl is the only specific thing to do, I agree we can remove this file and add a little word about it in the main readme.